src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -206,6 +211,13 @@ def source_waveform_type(request):
 
 
 def test_source_waveform_parallel_site_unique(multi_instrument_session, source_waveform_type):
+    '''Test methods for generating and capturing unique waveforms to multiple sites in parallel.
+
+    - create_capture_waveform_parallel
+    - create_source_waveform_parallel
+    - fetch_capture_waveform
+    - write_source_waveform_site_unique
```

Comment:
Remove create_capture and fetch_capture from this list. They are being used to validate the source feature.

Reply:
Done.